### PR TITLE
Feat/add integration test dcatview

### PR DESCRIPTION
**Pull request by Tomvbe**

### CODE REVIEW

File: ldes-server-port-admin-rest/src/test/resources/features/dcatview/dcatview.feature [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Scenario Outline: Delete DCAT Dataservice
+      Given I have a <isValid> dcat dataservice
+      When I DELETE this dataservice
+      Then The dataservice metadata will be deleted
+      And Response with http 200 will be returned
+
+      Examples:
+        | isValid |  |  |
+        | valid   |  |  |
+        | invalid |  |  |
```
Comment (Yalz) [inline ldes-server-port-admin-rest/src/test/resources/features/dcatview/dcatview.feature:38]: do you need the extra columns for this table?

**Review by Yalz — APPROVED** [review]

File: ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/entity/DcatView.java [hunk 2]
```diff
@@ -38,19 +40,17 @@ public Model getDcat() {
...
 
-		List<Statement> statements = getDcat().listStatements()
-				.mapWith(stmnt -> stmnt.getSubject().equals(dataServiceId)
-						? createStatement(viewDescriptionResource, stmnt.getPredicate(), stmnt.getObject())
-						: stmnt)
-				.toList();
+		final Model dcatWithIdentity = ModelFactory.createDefaultModel();
+		dcatWithIdentity.add(getDcat());
+		dcatWithIdentity.listStatements(null, RDF.type, DCAT_DATA_SERVICE).nextOptional()
+				.ifPresent(statement -> renameResource(statement.getSubject(), viewDescriptionResource.getURI()));
```
[the remark this answers is not in the document]
Comment (Tomvbe) [reply ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/entity/DcatView.java:48]: Yes, that is why I create a copy before renaming, to keep the original model intact

File: ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/DcatViewsRestController.java [hunk 3]
```diff
@@ -14,21 +14,23 @@
...
-@RequestMapping("/admin/api/v1/eventstreams/{collectionName}/views/{viewName}/dcat")
+@RequestMapping(DcatViewsRestController.BASE_URL)
 public class DcatViewsRestController implements OpenApiDcatViewsController {
 
+	public final static String BASE_URL = "/admin/api/v1/eventstreams/{collectionName}/views/{viewName}/dcat";
+
 	private final DcatViewService dcatViewService;
 
-	private final DcatViewValidator dcatViewValidator;
+	private final DcatDataServiceValidator dcatDataServiceValidator;
```
[the remark this answers is not in the document]
Comment (Tomvbe) [reply ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/DcatViewsRestController.java:24]: You are right, I will rename this.

File: ldes-server-port-admin-rest/src/test/resources/features/dcatview/dcatview.feature (hunk 1 above)
Comment (Tomvbe) [reply ldes-server-port-admin-rest/src/test/resources/features/dcatview/dcatview.feature:38]: Intellij didn't let me delete them, I was hoping that they wouldn't actually exist :(

**Pull request merged**